content/blogs/release-1-1.md

```diff
@@ -0,0 +1,359 @@
+---
+title: "Kestra 1.1 introduces New Filters, No-Code Dashboards, Human Tasks, AI Agent tool and Dozens of New Plugins"
+description: "Kestra 1.1 delivers a polished UI with redesigned filters and no-code dashboard editor, human-in-the-loop workflows for enterprise, improved airgap support, and dozens of new community-driven plugins."
+date: 2025-11-04T17:00:00
+category: News & Product Updates
+authors:
+  - name: "Benoit Pimpaud"
+    image: bpimpaud
+image: /blogs/release-1-1.jpg
+---
+
+
+We're excited to announce Kestra 1.1, delivering a polished UI experience, dozens of community-driven plugins, and powerful enterprise features. Following user feedback, we've completely redesigned filters, introduced a no-code dashboard editor, and enhanced airgap support for offline deployments. Enterprise Edition users gain custom branding capabilities and human-in-the-loop workflows with granular permission controls. This release also brings new integrations for Liquibase, dlt, YouTube, Stripe, and many more—connecting Kestra to your entire tech stack.
+
+The table below highlights the key features of this release.
+
+| Feature | Description | Edition |
+|---|---|---|
+| New Filters | Redesigned UI filters with faster autocompletion and improved usability based on user feedback | All Editions |
```

Comment:
```suggestion
| New Filters | Redesigned UI filters with improved usability based on user feedback | All Editions |
```
technically speaking, we're removing the editor with autocompletion so can be confusing